R/fd_belowground.R

```diff
@@ -21,10 +23,11 @@ fd_soil_respiration <- function() {
 
   # Timestamp
   flux$Timestamp <- as.POSIXct(flux$dateTime, format = "%m/%d/%Y %H:%M", tz = "America/Detroit")
+  flux$Date <- mdy(flux$Date)
```

Comment:
For this, it may or may not be worth importing `lubridate` (see previous comment). We could also just use base R's `strptime`.

Reply:
Also: given that we have `Timestamp` above, why bother with `Date` in this line at all?